build_windows.py

```diff
@@ -145,45 +148,46 @@
 		)
 	# Copy some theme icons
 	sizes = ["16x16", "24x24", "32x32", "scalable"]
-	icons = {	"status"  : [
-					"image-missing",
-					"dialog-information",
-					"dialog-warning",
-					"dialog-error",
-					"checkbox-symbolic",
-					"checkbox-mixed-symbolic",
-					"checkbox-checked-symbolic",
-				],
-				"mimetypes" :	[ "text-html" ],
-				"emblems" :		[ "emblem-system-symbolic" ],
-				"apps" :		[ "utilities-terminal" ],
-				"categories" :	[ "preferences-system" ],
-				"places":		[ "user-home" ],
-				"actions" : [
-					"help-about",
-					"edit-delete",
-					"edit-undo",
-					"application-exit",
-					"system-shutdown",
-					"document-open",
-					"view-refresh",
-					"open-menu-symbolic",
-					"window-close-symbolic",
-					"window-maximize-symbolic",
-					"window-restore-symbolic",
-					"window-minimize-symbolic",
-					"list-add-symbolic",
-					"list-remove-symbolic",
-					"pan-up-symbolic",
-					"pan-start-symbolic",
-					"pan-end-symbolic",
-					"pan-down-symbolic",
-				],
-				"devices" : [
-					"drive-harddisk",
-					"computer",
-				],
-		}
+	icons = {
+		"status"  : [
+			"image-missing",
+			"dialog-information",
+			"dialog-warning",
+			"dialog-error",
+			"checkbox-symbolic",
+			"checkbox-mixed-symbolic",
+			"checkbox-checked-symbolic",
+		],
+		"mimetypes" :	[ "text-html" ],
+		"emblems" :		[ "emblem-system-symbolic" ],
+		"apps" :		[ "utilities-terminal" ],
+		"categories" :	[ "preferences-system" ],
+		"places":		[ "user-home" ],
```

Comment:
The reason these look all weird and inconsistent is because it's mixing spaces and tabs - w/ tab width = 4 they all line up.

I assume your preference is tabs?  Technically speaking, pretty much every Python style guide recommends spaces, partly to prevent stuff like ^. (but I'm not going to press this or anything, it's your project).